### PR TITLE
prevent wrong asset model type assignment

### PR DESCRIPTION
**Pull request by szymi-**

### CODE REVIEW

File: src/ralph/back_office/admin.py [hunk 1]
```diff
@@ -58,7 +59,8 @@ class BackOfficeAssetLicenceInline(RalphTabularInline):
     inlines = [BackOfficeAssetLicenceInline]
 
 
-class BackOfficeAssetAdminForm(RalphAdmin.form):
+class BackOfficeAssetAdminForm(AssetFormMixin, RalphAdmin.form):
+    MODEL_TYPE = 'back_office'
```
Comment (ar4s) [inline src/ralph/back_office/admin.py]: use ``ObjectModelType.back_office`` instead of string

File: src/ralph/data_center/forms.py [hunk 2]
```diff
@@ -5,10 +5,12 @@
 
 from ralph.admin.mixins import RalphAdminForm
 from ralph.data_center.models.physical import DataCenterAsset
+from ralph.lib.mixins.forms import AssetFormMixin
 from ralph.networks.models import IPAddress
 
 
-class DataCenterAssetForm(RalphAdminForm):
+class DataCenterAssetForm(AssetFormMixin, RalphAdminForm):
+    MODEL_TYPE = 'data_center'
```
Comment (ar4s) [inline src/ralph/data_center/forms.py]: ditto

**Review by ar4s — CHANGES_REQUESTED** [review]

File: src/ralph/lib/mixins/forms.py [hunk 3]
```diff
@@ -94,3 +95,24 @@ def clean(self, value):
 
     def to_python(self, value):
         return value
+
+
+class AssetFormMixin(forms.ModelForm):
+    MODEL_TYPE = None
```
Comment (ar4s) [inline src/ralph/lib/mixins/forms.py:101]: IMO ``None`` is not valid value - consider some validation

**Review by ar4s — CHANGES_REQUESTED** [review]

File: src/ralph/lib/mixins/forms.py (hunk 3 above)
Comment (szymi-) [reply src/ralph/lib/mixins/forms.py:101]: What would you consider sufficient validation? Would checking if MODEL_TYPE is not None be enough?

**Review by ar4s — APPROVED** [review]

**Pull request merged**